docs/src/conceptual/solution-selection-catalogs.rst

```diff
@@ -145,28 +155,30 @@ Line **[D]** shows a child catalog for gfx906, similar to the gfx900 catalog. Ho
     type: Hardware                                           # [_A]
```

Comment:
@bstefanuk Should I remove underscore from _A, _B ..etc.? The reference given below has no underscore.